### PR TITLE
Added Htmlable Support For TableHeader.

### PR DESCRIPTION
**Pull request by Nuhel**
Always making view, and pass it as table header is not something we want all time as we can use htmlstring in filament.

So I add support for this.

Also i see there may be issue with table headeraction, So what i have done , If the table header pass as argument is Illuminate\Support\HtmlString, I just added the action with the heading.

Here is an demo what it look like.
Screenshot 2022-10-01 at 11 19 16 AM

Screenshot 2022-10-01 at 11 31 38 AM

### CODE REVIEW

**Comment by danharrin** [comment]
Your IDE has made hundreds of formatting changes, please revert them

**Comment by Nuhel** [comment]
> Your IDE has made hundreds of formatting changes, please revert them

Yes, Sorry for that.
Have a look now :)

File: packages/tables/resources/views/index.blade.php [hunk 1]
```diff
@@ -166,7 +166,18 @@ class="filament-tables-header-container"
             {!! ! $renderHeader ? 'x-cloak' : null !!}
         >
             @if ($header)
-                {{ $header }}
+                @if (gettype($header) == 'object' && get_class($header) == 'Illuminate\Support\HtmlString')
```
Comment (Z3d0X) [inline packages/tables/resources/views/index.blade.php:169]:
```suggestion
                @if (gettype($header) == 'object' && $header instanceof \Illuminate\Support\HtmlString)
```

**Comment by danharrin** [comment]
We should actually support Htmlable in the `getTableHeading()`, not `getTableHeader()`. You want to keep the action buttons, but thats not the behaviour of `getTableHeading()`.

**Comment by danharrin** [comment]
done

**Pull request merged**